.buildkite/scripts/steps/checks/saved_objects_compat_changes.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+
+export DISABLE_BOOTSTRAP_VALIDATION=false
+.buildkite/scripts/bootstrap.sh
+
+echo --- Check Mappings Update
+cmd="node scripts/check_mappings_update"
+if is_pr && ! is_auto_commit_disabled; then
+  cmd="$cmd --fix"
+fi
```

Comment:
I'm pretty sure this doesn't justify it's own worker (every worker takes several minutes to setup, short running tasks need to be manually combined with others), we should add this to the other checks in `.buildkite/scripts/steps/checks.sh` which are run in the "Checks" step.